src/renderer/components/fields/Checkbox.tsx

```diff
@@ -20,7 +20,7 @@ export const Checkbox: FC<ICheckbox> = ({
   visible = true,
   ...props
 }: ICheckbox) => {
-  const counter = props.counter > 0 ? props.counter : undefined;
+  const counter = props?.counter === 0 ? '0' : props.counter;
```

Comment:
When `0` is left as a number, it will not be rendered within the counter label correctly

Reply:
Sounds painful to figure out.

Worth leaving this as a code comment?